src/resources/InsightPanelInterfaces/InsightPanelInterface.model.ts

```diff
@@ -21,58 +16,6 @@ interface InsightPanelResultTagOptions {
     color: string;
 }
 
-export interface InsightPanelResultTemplateBadge {
```

Comment:
Here you're removing interfaces that were previously exported which is a breaking change. I'm not sure if that was your intention but if not then I suggest something like this instead:
```typescript
export interface InsightPanelResultTemplateBadge extends HostedInterfaceResultTemplateBadge {}
```
It makes sure the interface still respects the base interface but is non-breaking and allows for the "insight" version to actually be extended independently of the IPX version later if need be.
What do you think?